### DISCUSSION TITLE
Cannot Add Spacing To Wibar, Margins To Titlebar, and Cannot Change Titlebar Shape

### DISCUSSION THREAD

**Discussion by VehementHam** [awesomeWM/awesome#3870] [Q&A] [answered]
Here is my config: https://git.disroot.org/oink/dot-files/src/branch/main/Gentoo/home/vehementham/.config/awesome/rc.lua

I cannot figure how to add spacing to wibar, margins to titlebar, and change titlebar shape. I want the titlebar to have rounded corners.

**Comment by Elv13** [comment, marked as answer]
If you use Gentoo, you might want to switch to the 9999 version (git version) of AwesomeWM. [The documentation](https://awesomewm.org/apidoc/documentation/03-declarative-layout.md.html) is better. Then the answer to the first 2 question is to use `wibox.container.margin`. The shape one is that the shape is applied to the clients, not the titlebar. A titlebar is part of a client, it doesn't have its own shape.
